meeting_minutes/221027_SARIF_TC_69.md

```diff
@@ -0,0 +1,138 @@
+# 1. Opening Activities
+
+## 1.1 Opening comments (Co-Chair David)
+
+## 1.2 Introduction of participants/roll call (Co-Chair David)
+
+| First Name | Last Name | Company           | Role(s)                    |
+|:-----------|:----------|:------------------|:---------------------------|
+| Charles    | Wilson    | Motional AD       | Member                     |
+| David      | Keaton    | Individual        | Chair                      |
+| Johnathan  | Gilday    | Contrast Security | Voting Member              |
+| Mary       | Martin    | Microsoft         | Voting Member              |
+| Michael    | Fanning   | Microsoft         | Voting member, taking notes|
+| Nathan     | Baird     | Microsoft         | Voting Member              |
+| Pavel      | Mikula    | Sonar             | Voting Member              |
+| Thanassis  | Avgerinos | ForAllSecure Inc  | Voting Member              |
+| Yekaterina | O'Neil    | Micro Focus       | Voting member              |
+
+## 1.3 Procedures for this meeting (Co-Chair David)
+
+## 1.4 Approval of agenda (Co-Chair David)
+
+* https://www.oasis-open.org/apps/org/workgroup/sarif/download.php/70473/agenda_20221027.html
+
+Agenda was approved
+
+## 1.5 Approval of previous minutes (Co-Chair David)
+
+* [Minutes of 2022-10-13_Meeting #68](https://github.com/oasis-tcs/sarif-spec/blob/main/meeting_minutes/221013_SARIF_TC_68.md)
+
+Minutes were approved
+
+## 1.6 Review of action items and resolutions (Secretary Stefan)
+
+* ACTION on Michael to keep pushing to close on the wikipedia page
+  * ONGOING
+* ACTION on Mary to continue to develop fixes scenario
+  * COMPLETE
+* ACTION on Jeff to provide an initial seed page at github to foster the future roadmap / extension discussion
+  * ONGOING
+
+## 1.7 Identification of SARIF TC voting members (Co-Chair David)
+
+### 1.7.1 Prospective members attending their first meeting
+
+### 1.7.2 Members attaining voting rights at the end of this meeting
+
+### 1.7.3 Members losing voting rights if they have not joined this meeting by the time it ends
+Jeff Williams and Chris Wysopal
```

Comment:
Will Jeff Williams lose voting rights even when he has declared leave of absence for October below?
https://github.com/oasis-tcs/sarif-spec/blob/main/meeting_minutes/220915_SARIF_TC_66.md#175-members-who-have-declared-a-leave-of-absence

Reply:
I had the same question because I recalled this discussion around Jeff. I only found Aditya's request in email. Based on Pavel's find, I'll move Jeff Williams to the 'on sabbatical' section (but will wait for @dmk42's confirmation of course).

Reply:
Jeff changed his mind and decided not to declare a leave of absence because he wanted the option of attending if his schedule opened up.  (An official leave of absence declaration is required to happen on the mailing list.  The discussion during the meeting was just a courtesy.)